pkgs/applications/misc/pdfstudio/default.nix

```diff
@@ -49,7 +50,7 @@ in
       sha256 = "sha256-wQgVWz2kS+XkrqvCAUishizfDrCwGyVDAAU4Yzj4uYU=";
     };
     extraBuildInputs = [
-      libgccjit #for libstdc++.so.6 and libgomp.so.1
+      (lib.getLib gcc)  # for libstdc++.so.6 and libgomp.so.1
```

Comment:
Does `stdenv.cc.cc.lib` work here?

Reply:
Testing this now.

Had to fix [the openjdk issue](https://github.com/NixOS/nixpkgs/pull/212363) first.

Reply:
Yep, it does.